internal/bundler/shim.go

```diff
@@ -10,7 +10,7 @@ import { dirname as __agentuity_dirname } from 'path';
 import { readFileSync as __agentuity_readFileSync, existsSync as __agentuity_existsSync } from 'fs';
 
 const __filename = __agentuity_fileURLToPath(import.meta.url);
-const __dirname = __agentuity_dirname(__filename);
+const __ag_dirname = __agentuity_dirname(__filename);
```

Comment:
sorry don't change this one. we need this to be what it was. ignore code rabbit for this one